### ISSUE TITLE
Feature request: max size

### ISSUE THREAD

**Issue by Tuzongyu** [magnusvk/FlickrFaves#5]
Great utility, couldn't find anything else out there for quickly downloading favorites.

I was wondering if you could add the option to download something other than the maximum resolution? I have an number of photos that are particular large (ie 6000x4000) and I'd prefer to bulk download flickr's "large" resolution rather than "original" in this case.

Thanks!

**Comment by magnusvk** [comment]
Right; this should probably be relatively easy to add but I'm not really actively working on this right now so probably won't get to it. That being said, if you wanted to open a pull request I'd be happy to look at it. 😄
